model/relabel/relabel.go

```diff
@@ -192,24 +192,29 @@ func (re Regexp) String() string {
 // are applied in order of input.
 // If a label set is dropped, nil is returned.
 // May return the input labelSet modified.
-func Process(labels labels.Labels, cfgs ...*Config) labels.Labels {
+func Process(lbls labels.Labels, cfgs ...*Config) labels.Labels {
+	lb := labels.NewBuilder(nil)
```

Comment:
I'd like another interface called `ProcessWithBuilder`.

Reply:
OK; is that relevant to this PR?
If not, it would belong in a separate PR with its own justification.

Reply:
I was thinking about using a pool of Builders here.

Reply:
That could also be done inside `Process` without changing the api.